changelogs/fragments/5038-add-crc32-filter.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - filter - add ``crc32`` filter to create small string checksums.
```

Comment:
Please remove this. New filters do not need changelog fragments.

Reply:
File removed